src/lib/SVGMap/ZoomControls.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
+  export let zoomIn;
+  export let zoomOut;
+  export let zoomReset;
+  export let showReset = false;
+  /*
+   * Where the zoom control UI should be positioned.
+   * @type {"bottom-left" | "bottom-right" | "top-left" | "top-right"}
+   */
+  export let controlPosition;
+
+  $: top = controlPosition.includes("top");
+</script>
+
+<div class="zoom-controls {controlPosition}">
+  {#if showReset && top}
+    <button class="zoom-button zoom-reset" on:click={zoomReset}>
+      <svg
+        width="15"
+        height="15"
+        viewBox="0 0 15 15"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M0 0H6V2H2V6H0V0Z" fill="black" />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M15 0L15 6L13 6L13 2L9 2L9 -2.62268e-07L15 0Z"
+          fill="black"
+        />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M15 15L9 15L9 13L13 13L13 9L15 9L15 15Z"
+          fill="black"
+        />
+        <path
+          fill-rule="evenodd"
+          clip-rule="evenodd"
+          d="M0 15L-2.62268e-07 9L2 9L2 13L6 13L6 15L0 15Z"
+          fill="black"
+        />
+      </svg>
+    </button>
+  {/if}
+  <button class="zoom-button zoom-out" on:click={zoomOut}>
```

Comment:
consider making this disabled on max zoom

Reply:
good call here, added this in for both max and min zoom